services/movie_session.py

```diff
@@ -42,3 +44,13 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = (
```

Comment:
no need to store tickets in a variable that you will not reuse. just return result